model/flow/identifier.go

```diff
@@ -121,6 +121,18 @@ func MerkleRoot(ids ...Identifier) Identifier {
 	return root
 }
 
+func MerkleRootOfList(entities []IDWithFingerprint) Identifier {
+	var root Identifier
+	tree := merkle.NewTree()
+	for _, e := range entities {
+		id := e.ID()
+		tree.Put(id[:], e.Fingerprint())
+	}
+	hash := tree.Hash()
+	copy(root[:], hash)
+	return root
+}
```

Comment:
just a headsup or maybe a comment this Merkle Root computation is very different from the way we do it for exec storage (ledger) this one uses blake2 for hashing.

Reply:
As discussed, switched to list hashing